src/commands/blocks.ts

```
@@ -25,19 +26,19 @@ import { startLiveMonitoring } from './_blocks.live.ts';
 function formatBlockTime(block: SessionBlock, compact = false): string {
 	const start = compact
 		? block.startTime.toLocaleString(undefined, {
-				month: '2-digit',
-				day: '2-digit',
-				hour: '2-digit',
-				minute: '2-digit',
-			})
+			month: '2-digit',
```

ESLint

[error] Expected indentation of 4 tabs but found 3.

(style/indent)